src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/DefaultJudgmentRepository.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.cmc.claimstore.repositories;
+
+import org.skife.jdbi.v2.sqlobject.Bind;
+import org.skife.jdbi.v2.sqlobject.GetGeneratedKeys;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+import org.skife.jdbi.v2.sqlobject.customizers.RegisterMapper;
+import org.skife.jdbi.v2.sqlobject.customizers.SingleValueResult;
+import uk.gov.hmcts.cmc.claimstore.models.DefaultJudgment;
+import uk.gov.hmcts.cmc.claimstore.repositories.mapping.DefaultJudgmentMapper;
+
+import java.util.Optional;
+
+@RegisterMapper(DefaultJudgmentMapper.class)
+public interface DefaultJudgmentRepository {
+
+    @SingleValueResult
+    @SqlQuery("SELECT * FROM default_judgment WHERE id = :defaultJudgmentId")
+    Optional<DefaultJudgment> getById(@Bind("defaultJudgmentId") Long defaultJudgmentId);
+
+    @SingleValueResult
+    @SqlQuery("SELECT * FROM default_judgment WHERE claim_id = :claimId")
+    Optional<DefaultJudgment> getByClaimId(@Bind("claimId") Long claimId);
+
+    @GetGeneratedKeys
+    @SqlUpdate("INSERT INTO default_judgment ( "
```

Comment:
I don't think it should be called default_judgment that's not a term used anywhere else.
county_court_judgment?

Reply:
ok